src/boot/bootctl-util.h

```diff
@@ -8,3 +8,11 @@ const char *get_efi_arch(void);
 int get_file_version(int fd, char **ret);
 
 int settle_entry_token(void);
+
+static inline const char* get_conf_root(void)
+{
+        const char* conf_root = getenv("KERNEL_INSTALL_CONF_ROOT");
+        if (conf_root)
+                return conf_root;
+        return "/etc/kernel/";
+}
```

Comment:
that bootctl interprets this env var should be documented in docs/ENVIRONMENT.md